static/styles/zulip.scss

```diff
@@ -566,6 +566,11 @@ td.pointer {
     @include prefixed-user-select(none);
 }
 
+.message_edit_notice:hover {
+    opacity: 1.0;
+}
+
+
```

Comment:
This should probably be a single newline. I'd suggest running `./tools/lint --frontend` locally on your dev env to spot and rectify linting issues like this one.

Reply:
Ok
Should i now make the changes to make it a single newline??